Covid/Services/LocationServices/BeaconManager.swift

```diff
@@ -61,10 +62,10 @@ final class BeaconManager: NSObject {
     override private init() {
         super.init()
 
-        locationManager.requestAlwaysAuthorization()
-        locationManager.desiredAccuracy = kCLLocationAccuracyHundredMeters
-        locationManager.allowsBackgroundLocationUpdates = true
         locationManager.delegate = self
+        if (Firebase.remoteConfig?.configValue(forKey: "ibeaconLocationAccuracy").numberValue ?? -1) != -1 {
```

Comment:
tento zapis budeme musiet v buducnosti vytunit :) 

idealne by sme zriesili eg enum pre keys, ktore registrujeme v remote konfigu, aby sme potom mohli pohodlne volat apinu ako napr.:

```
Firebase.value(for: . ibeaconLocationAccuracy).numberValue ...   
```

ci nieco podobne ;)

Reply:
presne nejaky enum, tieto keys, to uz len cakam kde sa pomylim, to je tak ked sa to snazis robit rychlo rychlo, mas to uz na 5 miestach a potom na kazde dalsie sa ti to nechce upravovat, ze by to bolo vela robboty a potom to mas zrazu na 30 miestach a uz sa ti to nechce vobec prepisovat, ale grcas ked to zapisujes 31. krat :)))

Reply:
haha, presne, postupne skonvergujeme spat k 10 vyskytom ;)